packages/kit/src/runtime/client/renderer.js

```diff
@@ -658,6 +658,14 @@ export class Renderer {
 								context: node_loaded.context
 							});
 
+							if (error_loaded?.loaded?.redirect) {
```

Comment:
`?.` isn't supported in Node 12, which is a platform we still support, so I'd write it the longhand way for now

Reply:
oh, I missed that this is client code. still iOS 13 doesn't support it and I don't see it used elsewhere in `runtime/client`, so I think I'd still avoid it